wallet/package.json

```diff
@@ -0,0 +1,200 @@
+{
+  "name": "@agoric/wallet-ui",
```

Comment:
We've also been calling this the "wallet app". wallet.agoric.app is where you get it.

"wallet-ui" or "wallet-app" both work for me but whichever you pick please ensure the workspace (parent path) matches the name here.